### PR TITLE
[#3593] Update translation strings fixed

### PR DESCRIPTION
**Pull request by muloem**
#### Before the PR (what is the issue or what needed to be done)
* Broken encoding in translation strings

#### The solution
* Fix encoding issues

#### Reviewer Checklist
* [ ] Added an explanation about the work done
* [ ] Connected the PR and the issue on Zenhub
* [ ] Added a test plan to the issue
* [ ] Updated the copyright header (when relevant)
* [ ] Formatted the code
* [ ] Added a documentation (if relevant)
* [ ] Added some unit tests (if relevant)

### CODE REVIEW

File: GAE/src/locale/es.properties [hunk 1]
```diff
@@ -1,141 +1,136 @@
...
 Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ give\ the\ survey\ a\ title = Antes de guardar o publicar una encuesta, debe seleccionar un tipo de encuesta
 Beginning\ of\ time = Desde el principio
-Build\ chart = Construir gráfico
+Build\ chart = Construir gr�fico
 Bulk\ upload\ data = Subir datos masivamente
-By\ repeating\ this\ question\ group\ you\ can\ ask\ questions\ in\ this\ group\ multiple\ times,\ e.g.\ when\ you\ want\ to\ ask\ the\ same\ questions\ to\ each\ member\ of\ a\ household.\ You\ will\ decide\ on\ the\ number\ of\ repetitions\ when\ capturing\ the\ answers\ in\ the\ Akvo\ Flow\ app.\ You\ cannot\ make\ a\ question\ outside\ of\ this\ group\ dependent\ on\ a\ question\ inside\ the\ group. = Al repetir este grupo de preguntas, puede hacer preguntas en este grupo varias veces, por ejemplo cuando desee hacer las mismas preguntas a cada miembro de un hogar. Usted decidirá el número de repeticiones al capturar las respuestas en la aplicación Akvo Flow. Una pregunta de otro grupo no puede depender de una pregunta de este grupo.
+By\ datapoint\ name\ or\ ID = By datapoint name or ID
```
Comment (valllllll2000) [inline GAE/src/locale/es.properties:70]: This one seems to be in english

Comment (muloem) [inline GAE/src/locale/es.properties:70]:
```suggestion
By\ datapoint\ name\ or\ ID = Asignar a través de nombre o ID del punto de datos
```

File: GAE/src/locale/es.properties [hunk 2]
```diff
@@ -70,13 +67,12 @@ Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ give\ the\ survey\ a\
 Beginning\ of\ time = Desde el principio
 Build\ chart = Construir gráfico
 Bulk\ upload\ data = Subir datos masivamente
+By\ datapoint\ name\ or\ ID = Asignar a través de nombre o ID del punto de datos
```
Comment (muloem) [inline GAE/src/locale/es.properties:70]:
```suggestion
By\ datapoint\ name\ or\ ID = Asignar por nombre o ID del punto de datos
```

File: GAE/src/locale/es.properties [hunk 3]
```diff
@@ -70,13 +67,12 @@ Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ give\ the\ survey\ a\
 Beginning\ of\ time = Desde el principio
 Build\ chart = Construir gráfico
 Bulk\ upload\ data = Subir datos masivamente
+By\ datapoint\ name\ or\ ID = Asignar por nombre o ID del punto de datos
```
Comment (muloem) [inline GAE/src/locale/es.properties:70]:
```suggestion
By\ datapoint\ name\ or\ ID = Por nombre o ID del punto de datos
```

File: GAE/src/locale/es.properties [hunk 4]
```diff
@@ -145,154 +140,154 @@ Delete\ form = Borrar formulario
...
 Manage\ device\ groups = Administrar grupos de dispositivos
 Manage\ notifications = Administrar notificaciones
 Manage\ translations = Administrar traducciones
 Mandatory = Obligatorio
 Manual\ survey\ transfer = Transferencia manual de encuesta
 Maps = Mapas
-Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marcar respuestas a esta pregunta como datos personales (datos que se pueda usar para identificar a un individuo). Al exportar los datos a travs del Flow API, se puede filtrar los datos con en este indicador
+Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marcar respuestas a esta pregunta como datos personales (datos que se pueda usar para identificar a un individuo). Al exportar los datos a trav\u008es del Flow API, se puede filtrar los datos con en este indicador
```
Comment (valllllll2000) [inline GAE/src/locale/es.properties:263]: trav\u008es: this is a bit strange, I see all the other chars represented correctly, ie í, ó shouldn't it be "través"?

File: GAE/src/locale/es.properties [hunk 5]
```diff
@@ -265,7 +260,7 @@ Manage\ translations = Administrar traducciones
...
```
Comment (muloem) [inline GAE/src/locale/es.properties:263]:
```suggestion
Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marcar respuestas a esta pregunta como datos personales (datos que se pueda usar para identificar a un individuo). Al exportar los datos a través del Flow API, se puede filtrar los datos con en este indicador
```

File: GAE/src/locale/fr.properties [hunk 6]
```diff
@@ -21,269 +21,264 @@ Add\ new\ question = Ajouter une question
...
+Before\ you\ can\ add\ questions\ groups\ to\ a\ survey,\ you\ first\ need\ to\ save\ the\ survey.\ click\ ok\ the\ save\ button\ in\ the\ left\ sidebar = Avant de pouvoir ajouter des groupes de questions � une enqu�te, vous devez tout d'abord enregistrer cette enqu�te. Cliquez sur ok le bouton 'Sauvegarder' dans la barre lat�rale gauche
+Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ give\ the\ survey\ a\ title = Avant de publier ou d'enregistrer une enqu�te, vous devez donner un titre � celle-ci.
+Beginning\ of\ time = D�but du temps
+Build\ chart = G�n�rer le graphique
+Bulk\ upload\ data = Importer donn�es manuellement 
+By\ datapoint\ name\ or\ ID = By datapoint name or ID
```
Comment (muloem) [inline GAE/src/locale/fr.properties:70]:
```suggestion
By\ datapoint\ name\ or\ ID = Par nom ou ID des points de données
```

File: GAE/src/locale/pt.properties [hunk 7]
```diff
@@ -1,550 +1,542 @@
...
 Save\ question = Gravar pergunta
-Save\ translations = Salvar traduções
-Save\ user\ info = Gravar informações do usuário
+Save\ translations = Salvar tradu��es
+Save\ user\ info = Gravar informa��es do usu�rio
 Saving = Gravando
-Secret = Sigilo
-Select\ a\ folder\ or\ survey = Seleccionar uma pasta ou Inquérito
-Select\ a\ role = Seleccionar função
+Search\ datapoint\ by\ name\ or\ ID,\ or\ press\ Enter\ to\ show\ all = Search datapoint by name or ID, or press Enter to show all
```
Comment (muloem) [inline GAE/src/locale/pt.properties:393]:
```suggestion
Search\ datapoint\ by\ name\ or\ ID,\ or\ press\ Enter\ to\ show\ all = Pesquise o ponto de dados por nome ou ID ou pressione Enter para mostrar tudo
```

File: GAE/src/locale/pt.properties [hunk 8]
```diff
@@ -48,30 +46,28 @@ Approval\ status = Estado de aprovação
...
 Before\ publishing,\ you\ have\ to\ save\ your\ changes\ by\ clicking\ the\ "save"\ button = Antes de publicar, deve gravar as suas alterações clicando em "Gravar"
-Before\ saving\ an\ assignment,\ you\ need\ to\ set\ an\ assignment\ name = Antes de gravar um emparelhamento, você precisa definir um nome
 Before\ you\ can\ add\ questions\ groups\ to\ a\ survey,\ you\ first\ need\ to\ save\ the\ survey.\ click\ ok\ the\ save\ button\ in\ the\ left\ sidebar = Antes de adicionar grupos de perguntas num questionário, deve primeiro gravar o inquérito. Clique em Aprovar na barra lateral
 Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ give\ the\ survey\ a\ title = Antes de gravar ou publicar um inquérito, deve criar um título para ele.
-Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ select\ a\ survey\ type = Antes de gravar ou publicar um inquérito, deve seleccionar um tipo de inquérito
+Beginning\ of\ time = Beginning of time
 Build\ chart = Criar gráfico
 Bulk\ upload\ data = Carregar dados em massa 
+By\ datapoint\ name\ or\ ID = By datapoint name or ID
```
Comment (muloem) [inline GAE/src/locale/pt.properties:70]:
```suggestion
By\ datapoint\ name\ or\ ID = Por nome ou ID o ponto de dados
```

File: GAE/src/locale/pt.properties [hunk 9]
```diff
@@ -1,37 +1,35 @@
...
 A\ survey\ can\ only\ be\ created\ inside\ a\ folder = Uma pesquisa só pode ser criada dentro de uma pasta
-API\ keys = Chaves API
-Access\ key = Chave de acesso
 Action = Acção
 Action\ failed = A acção falhou
 Actions = Acções
 Add = Adicionar
 Add\ Step = Adicionar etapa
 Add\ approval\ group = Adicionar grupo de aprovação
+Add\ devices\ to\ assignment = Adicionar dispositivos ao grupo de dispositivos
```
Comment (valllllll2000) [inline GAE/src/locale/pt.properties:13]: This is not right translation, should be: Adicionar dispositivos à atribuição

File: GAE/src/locale/pt.properties [hunk 10]
```diff
@@ -1,37 +1,35 @@
...
 Add\ new\ form = Adicionar novo formulário\n
-Add\ new\ metric = Adicionar nova métrica
 Add\ new\ option = Adicionar nova opção
 Add\ new\ question = Adicionar nova questão
 Add\ new\ role = Adicionar nova função
 Add\ new\ translation = Adicionar nova tradução
 Add\ new\ user = Adicionar novo usuário
-Add\ selected\ devices = Adicionar dispositivos seleccionados
 Add\ selected\ forms = Adicionar formulários selecionados
+Add\ to\ assignment = Add to assignment
```
Comment (valllllll2000) [inline GAE/src/locale/pt.properties:25]: Should be: Adicionar à atribuição

File: GAE/src/locale/pt.properties [hunk 11]
```diff
@@ -1,37 +1,35 @@
...
 Add\ to\ device\ group = Adicionar ao grupo de dispositivos
 Administrator\ -\ Create,\ Modify\ and\ Delete\ Users = Administrador - Criar, Modificar e Apagar usuários
 Akvo\ Caddisfly = Akvo Caddisfly 
 All\ folders = Todas pastas
 All\ folders\ and\ surveys = Todas pastas e inquéritos
+All\ options\ codes\ must\ have\ values = Todos os códigos de opções devem ter valores
+All\ {}\ datapoints\ assigned = All {} datapoints assigned
```
Comment (valllllll2000) [inline GAE/src/locale/pt.properties:32]: should be: Todos os {} pontos de dados atribuídos

File: GAE/src/locale/pt.properties (hunk 10 above)
Comment (muloem) [inline GAE/src/locale/pt.properties:25]:
```suggestion
Add\ to\ assignment = Adicionar à atribuição
```

File: GAE/src/locale/pt.properties (hunk 9 above)
Comment (muloem) [inline GAE/src/locale/pt.properties:13]:
```suggestion
Add\ devices\ to\ assignment = Adicionar dispositivos à atribuição
```

File: GAE/src/locale/pt.properties [hunk 12]
```diff
@@ -245,57 +246,65 @@ Length = Comprimento
...
 Manage\ translations = Gerir traduções
-Manage\ users\ and\ user\ rights = Gerir suários e direitos do usúrio
 Mandatory = Obrigatório
 Manual\ survey\ transfer = Transferência manual do inquérito 
 Maps = Mapas
-Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marque as respostas para esta pergunta como dados pessoais (quaisquer dados que possam ser usados ​​para identificar um indivíduo). Ao exportar dados por meio da API de Flow, você pode excluir respostas com base nesse indicador
+Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marque as respostas para esta pergunta como dados pessoais (quaisquer dados que possam ser usados \u200b\u200bpara identificar um indivíduo). Ao exportar dados por meio da API de Flow, você pode excluir respostas com base nesse indicador
```
Comment (valllllll2000) [inline GAE/src/locale/pt.properties:263]: replace \u200b\u200bpara by para

File: GAE/src/locale/pt.properties [hunk 13]
```diff
@@ -430,21 +420,24 @@ Show\ questions = Mostrar perguntas
 Show\ shape\ on\ map = Mostrar a forma no mapa
 Signature = Assinatura
 Signed\ by = Assinado por
+Similar\ options\ can't\ be\ defined\ multiple\ times = Opções semelhantes não podem ser definidas várias vezes
+Similar\ options\ codes\ can't\ be\ used\ multiple\ times = Códigos de opções semelhantes não podem ser usados \u200b\u200bvárias vezes
```
Comment (valllllll2000) [inline GAE/src/locale/pt.properties:424]: remove \u200b\u200b

File: GAE/src/locale/pt.properties (hunk 10 above)
Comment (muloem) [inline GAE/src/locale/pt.properties:25]:
```suggestion
Add\ to\ assignment = Adicionar à atribuição
```

File: GAE/src/locale/pt.properties (hunk 11 above)
Comment (muloem) [inline GAE/src/locale/pt.properties:32]:
```suggestion
All\ {}\ datapoints\ assigned = Todos os {} pontos de dados atribuídos
```

File: GAE/src/locale/pt.properties (hunk 12 above)
Comment (muloem) [inline GAE/src/locale/pt.properties:263]:
```suggestion
Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marque as respostas para esta pergunta como dados pessoais (quaisquer dados que possam ser usados para identificar um indivíduo). Ao exportar dados por meio da API de Flow, você pode excluir respostas com base nesse indicador
```

File: GAE/src/locale/pt.properties (hunk 13 above)
Comment (muloem) [inline GAE/src/locale/pt.properties:424]:
```suggestion
Similar\ options\ codes\ can't\ be\ used\ multiple\ times = Códigos de opções semelhantes não podem ser usados várias vezes
```

**Pull request merged**